medical/gql_mutations.py

```diff
@@ -131,12 +128,54 @@ def update_or_create_item_or_service(data, user, item_service_model):
         check_if_code_already_exists(data, item_service_model)
 
     if item_service_uuid:
+        item_service = item_service_model.objects.get(uuid=item_service_uuid)
+        # Delete Service present in the Database and absent in the list sent by FE
+        # Means that user click on delete button and old Service is not sent
+        serviceExisting = list()
+        serviceSent = list()
+        for ServiceList in ServiceService.objects.filter(servicelinkedService=item_service.id).all() :
```

Comment:
no validity check here
```suggestion
        for ServiceList in ServiceService.objects.filter(*filter_validity(),servicelinkedService=item_service.id).all() :
```